src/utils/utils_log.c

```diff
@@ -156,26 +156,23 @@ void util_log_init(void) {
             len = strlen(arg);
         }
 
-        if (len <= MAX_FILE_PATH) {
-            memcpy(file, arg, len);
-            file[len] = '\0';
-#ifdef _WIN32
-            if (fopen_s(&loggerConfig.output, file, "w")) {
-                loggerConfig.output = NULL;
-            }
-#else
-            loggerConfig.output = fopen(file, "w");
-#endif
+        if (len > MAX_FILE_PATH) {
+            loggerConfig.output = stderr;
+            LOG_ERR("Cannot open output file - path too long", file);
```

Comment:
There's no string format specifier `%s` to log the `file` variable.

Reply:
Done